scripts/firedrake-configure

```diff
@@ -363,8 +375,10 @@ def prepare_external_package_configure_options(
             # Package is installed but not findable, have to provide paths
             # to headers and libraries
             (include_dir, libs) = package_spec
-            configure_options.append(f"--with-{external_package}-include={include_dir}")
-            configure_options.append(f"--with-{external_package}-lib={','.join(libs)}")
+            if include_dir != "":
```

Comment:
I think `None` is a more intuitive null value here.

Reply:
Agreed, done.